components/HeadingComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentConfig } from "@measured/puck";
+import React from "react";
+
+export type HeadingComponentProps = {
+  text: string,
+  textAlign: any,
+};
+
+function HeadingComponent({text, textAlign}: HeadingComponentProps){
+  return ( 
+    <div style={{padding: 64}}>
```

Comment:
Is this padding needed, or is there a way to use the margin and min-height on the h1 tag?